### PR TITLE
Commit RAC-3711-change-ora-references

### PR DESCRIPTION
**Pull request by gpaulos**
This PR just renames 'ora' to 'rackhd_host' in reference to the appliance hostname or IP address for RAC-3711.

Tested on deployments and smoke test.

Please merge https://eos2git.cec.lab.emc.com/OnRack/dellemc-test/pull/7 after this PR.

@johren @hohene @jimturnquist @larry-dean

### CODE REVIEW

**Comment by gpaulos** [comment]
@johren @hohene @jimturnquist @larry-dean 
Please review.

File: test/common/fit_common.py [hunk 1]
```diff
@@ -275,13 +275,13 @@ def mkargs(in_args=None):
...
-                            help="stack label (test bed), overrides -ora")
-    arg_parser.add_argument("-ora", default="localhost",
-                            help="OnRack/RackHD appliance IP address or hostname, default: localhost")
-    arg_parser.add_argument("-version", default="onrack-devel",
-                            help="OnRack package install version, example:onrack-release-0.3.0, default: onrack-devel")
+                            help="stack label (test bed), overrides -rackhd_host")
+    arg_parser.add_argument("-rackhd_host", default="localhost",
+                            help="RackHD appliance IP address or hostname, default: localhost")
+    arg_parser.add_argument("-version", default="None",
+                            help="RackHD install version, not yet implemented")
```
Comment (johren) [inline test/common/fit_common.py]: Can we just remove this option for now and add it when/if it gets implemented?  It should be pretty easy to add back in and would avoid confusion in the meantime.

Comment (gpaulos) [inline test/common/fit_common.py]: OK

**Comment by gpaulos** [comment]
Now I have a conflct in the README file.

**Comment by gpaulos** [comment]
Reverted README to base branch to fix conflict. README still needs revision. I will do that in another PR.

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
PR stale, needs to be rebased and resubmitted. Closing for resubmit.

**Pull request closed without merging**